hazelcast/src/test/java/com/hazelcast/config/ConfigXmlGeneratorTest.java

```diff
@@ -196,14 +199,144 @@ public void testRingbuffer() {
         assertEquals(rbConfig, xmlRbConfig);
     }
 
+    @Test
```

Comment:
Multiple tests were missing completely. I added them and created an issue to cover the missing attributes.
https://github.com/hazelcast/hazelcast/issues/12119